docs/source/install.rst

```diff
@@ -17,8 +17,9 @@ Change your directory to ``neuralmonkey`` folder::
 
 	cd neuralmonkey
 
-And now run pip to install all requirements. For CPU version install dependencies
-by this command::
+And now run pip to install all requirements. Try to avoid using --upgrade if
+you already have installed older version of packages.
```

Comment:
To nechceš, když budou mít něco older (třeba ten python-magic), tak jim to nebude fungovat.
Dej tam:

If you don't want to upgrade your packages, for example if you installed Tensorflow from sources and do not want to replace it with a downloaded package, consult the requirements file and install the dependencies by hand.